src/Core/Resolvers/Sql Query Structures/BaseSqlQueryStructure.cs

```diff
@@ -346,8 +346,20 @@ protected List<LabelledColumn> GenerateOutputColumns()
         /// <param name="systemType"></param>
         /// <returns></returns>
         /// <exception cref="NotSupportedException"></exception>
-        protected static object ParseParamAsSystemType(string param, Type systemType)
+        protected object ParseParamAsSystemType(string param, Type systemType)
         {
+            if (systemType.Name.Equals("DateTime"))
```

Comment:
Can we just make this 

`if (systemType.Name.Equals("DateTime") && ApitType is ApiType.GraphQL)`

and then include the `return DateTime.Parse(param)` in the below switch case?